### ISSUE TITLE
Not Planting Anything

### ISSUE THREAD

**Issue by CardinalBirb** [prinzstani/CookieBot#67]
CookieBot doesn't seem to be planting anything in the garden, even though it says it's trying to get Wheat Slims. Garden planting has been a bit jagged before it too. It doesn't seem to be clearing plants either when I tried planting non-baker's-wheat plants. 

Previously I was planting things manually when I noticed spots lacking the necessary plants for crossbreeding. Is it because I was manually messing with the garden that this is now happening?

[not planting anything save file.txt](https://github.com/prinzstani/CookieBot/files/6815743/not.planting.anything.save.file.txt)

**Comment by CardinalBirb** [comment]
It just cleared my field of Thumbcorn as a test but didn't plant anything. The bot also does seem to know to switch dirts, just not plant.

**Comment by CardinalBirb** [comment]
Bot seems to be removing mature plants just fine but is not planting anything. I tried planting a field of Baker's What and it harvested pretty quickly after reaching maturity.

**Comment by prinzstani** [comment]
Yes, I noticed this myself. The reason is that the bot is trying to avoid expensive plants, so it does not buy plants as long as the cps is high (much more than normal). It seems you have enabled cheating, such that the cps has always at least a factor of 3.5, which is considered too much. There is some kind of workaround in the current beta, which avoids this in case you do the cheating and which in any case makes a note in the status that plants are too expensive.
A more long-term fix is not too difficult but beyond my current capacity.
Maybe in autumn.

**Comment by CardinalBirb** [comment]
I see, thank you.
